### ISSUE TITLE
Drop shadow `filterSize` argument not recognized

### ISSUE THREAD

**Issue by talsafran** [airbnb/lottie-web#2179]
Hi there! Loving this library 😻 

I've seen several issues here about how to use drop shadows, and they all seem to be referencing the [`filterSize`](https://github.com/airbnb/lottie-web/wiki/Renderer-Settings#filtersize-svg-renderer) property for the SVG renderer:

```javascript
var animation = lottie.loadAnimation({
  ...
  filterSize: {
    width: '200%',
    height: '200%',
    x: '-50%',
    y: '-50%',
  }
})
```

I'm using TypeScript and am seeing an issue that the `filterSize` property is not an allowed property:

```
Argument of type '{ autoplay: true; container: Element; filterSize: { width: string; height: string; x: string; y: string; }; loop: true; path: string | undefined; renderer: "svg"; }' is not assignable to parameter of type 'AnimationConfigWithPath | AnimationConfigWithData'.
  Object literal may only specify known properties, and 'filterSize' does not exist in type 'AnimationConfigWithPath | AnimationConfigWithData'.ts(2345)
```

I looked into the `AnimationConfig` definition and `filterSize` doesn't seem to appear as a valid argument}:

https://github.com/airbnb/lottie-web/blob/411a0d9a2072f263ae6274f6a042f919fd9ec86f/index.d.ts#L78-L86

I looked at `SVGRendererConfig` also:

https://github.com/airbnb/lottie-web/blob/411a0d9a2072f263ae6274f6a042f919fd9ec86f/index.d.ts#L56-L65

I actually couldn't find any mention of `filterSize` in the repo 😄. Did something change? What's the proper way to get drop shadows to show up?

Thank you!

**Comment by bodymovin** [comment]
Hi, filterSize was missing from Typescript definitions. It'll be added on the next build.
Probably by the end of this weekend.

**Comment by talsafran** [comment]
Hey @bodymovin 👋 

I opened https://github.com/airbnb/lottie-web/pull/2181, which I think should resolve the issue. Would love your eyes when you have a chance!

**Comment by talsafran** [comment]
Just bumping this thread 🤗. I think I have a fix ready to go in #2181.

**Comment by bodymovin** [comment]
Thanks! I'll deploy a new build this weekend
